magnetic/Glk/glk.c

```diff
@@ -5168,11 +5215,13 @@ gms_expand_abbreviations (char *buffer, int size)
       memmove (command + strlen (expansion) - 1, command, strlen (command) + 1);
       memcpy (command, expansion, strlen (expansion));
 
+#if 0
```

Comment:
Is this correct?
This change comes from [Garglk](https://github.com/garglk/garglk/blob/302847873ee83fb8f87c15000dc05117c8f8e3a8/terps/magnetic/Glk/glk.c#L5216)